app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb

```diff
@@ -7,6 +7,73 @@ def available_vms
     availability_zone.vms
   end
 
+  def self.params_for_create(ems_id)
+    {
+      :fields => [
+        {
+          :component => 'switch',
+          :name      => 'shareable',
```

Comment:
Is this field backed by a DB column?

Reply:
Shareable here just means that the volume can be attached to more than one VM, so I _think_ the `multi_attachment` column fits.

Reply:
@agrare can you confirm this?

Reply:
Openstack Cinder refresh sets `multi_attachment`, should the powervs parser also collect&set this property?
Also could we match the column name with the DDF forms?

Reply:
> should the powervs parser also collect&set this property?
Yes - I can submit a PR for this

> Also could we match the column name with the DDF forms?
Yes, but I'd like to keep the label as `Shareable` to match the PowerVS web interface.

Reply:
:+1:

Reply:
The label can stay `shareable` but the name/id should be matching the DB colum, which I changed...